packages/server/lib/config.coffee

```diff
@@ -204,13 +216,45 @@ validateFile = (file) ->
     validate settings, (errMsg) ->
       errors.throw("SETTINGS_VALIDATION_ERROR", file, errMsg)
 
+validateBrowserList = (browsers) ->
+  # use try / catch inside to make sure we report the error
+  # and don't swallow it during integration testing
+  # otherwise I had situations where a test was failing in a very weird
+  # ways because an error was thrown here but the test continued
```

Comment:
curious about this, which spec was failing?

Reply:
don't remember - just remember silence, might be because we were catching errors in tests, but I want this error to be discoverable

Reply:
i don't see how `throw err` differs from `try { throw err } catch (err) { throw err }`, are you 100% sure this is necessary?

Reply:
it is NOT throwing the error - the error propagates. It is that we don't see that the invalid browser was the source of error, which allows one to chase wild guesses, rather than the true reason for this error

Reply:
ah, so this attaches the proper stack trace?

Reply:
no, it is still the same, I believe - but this prints the error before throwing it again, allowing you to see it

Reply:
must've missed that debug statement. got it, thanks!